fairscale/experimental/nn/ssd_offload.py

```diff
@@ -264,7 +285,7 @@ def get_tensors(self) -> List[SsdTensorHandle]:
 
     def to_disk(self) -> None:
         """Writes all tensors backed by handles to disk."""
-        assert list(self.buffer.size()) != [1]
+        assert self.storage_state == StorageState.IN_MEMORY
         # TODO(anj): Add comment about why we use `narrow`.
```

Comment:
We use narrow so the write is only for the valid tensors. If the SSDBuffer is allocated with a large 10MB buffer, but we only allocate 1MB in tensors, we don't want to write all 10MB to disk, only the 1MB that is valid.